### PR TITLE
Fix projectId for fetching build artifact.

### PR DESCRIPTION
**Pull request by omeshp**

### CODE REVIEW

File: src/Agent.Worker/Release/Artifacts/BuildArtifact.cs [hunk 1]
```diff
@@ -99,6 +99,10 @@ public IArtifactDetails GetArtifactDetails(IExecutionContext context, AgentArtif
             var tfsUrl = context.Variables.Get(WellKnownDistributedTaskVariables.TFCollectionUrl);
 
             Guid projectId = context.Variables.System_TeamProjectId ?? Guid.Empty;
+            if(artifactDetails.ContainsKey("Project")){
```
Comment (GitHubSriramB) [inline src/Agent.Worker/Release/Artifacts/BuildArtifact.cs]: space after if

Comment (GitHubSriramB) [inline src/Agent.Worker/Release/Artifacts/BuildArtifact.cs]: use style we use in c# for if block (this one resembles TS files)

if (XXX)
{
...
}

Comment (omeshp) [inline src/Agent.Worker/Release/Artifacts/BuildArtifact.cs]: there is.

**Comment by omeshp** [comment]
https://github.com/Microsoft/vsts-tasks/issues/3804

**Pull request merged**